### PR TITLE
[DE] Add `Beatmap/Title text`

### PR DESCRIPTION
**Pull request by FreezyLemon**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by The-Last-Cookie** [comment]
- [ ] Requesting self-review. I already began with the review in the PR, though I want to take another look later.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Title_text/de.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- `time` wird zu `(time aus vorheriger Zeile) + (wait aus vorheriger Zeile) + 200`
+- `hold` wird zu `(hold aus vorheriger Zeile) - 200`
+
+Wenn `time` explizit festgelegt wird, wird `hold` auf `4000 + 800n` gesetzt. Aus diesem Grund ist es wichtig, den Wert für `time` immer *vor* dem Wert für `hold` festzulegen, wenn beide für dieselbe Zeile festgelegt werden. Ansonsten wird `hold` überschrieben.
+
+#### Beispiel
+
+`[time:600,size:30,colour:255.0.0]Zeile 1|[bold:0,wait:100,hold:2000]Zeile 2`
+
+"Zeile 1" fängt ab dem Zeitpunkt 1100 ms an, eingeblendet zu werden (als Erinnerung: `wait` verzögert die Startzeit um 500 ms, wenn nicht anders festgelegt). Die Zeile wird in rot, fett und mit Schriftgröße 30 pt dargestellt. 5600 ms (`4000 + 800 * 2`) später fängt die Zeile an, ausgeblendet zu werden. "Zeile 2" fängt ab dem Zeitpunkt 1400 ms (1100 ms vom Start der vorherigen Zeile plus 200 ms, weil folgende Zeile plus 100 ms von `wait`) an, eingeblendet zu werden. Diese Zeile wird in weiß, nicht fett und mit Schriftgröße 40 pt dargestellt. 2000 ms später fängt die Zeile an, ausgeblendet zu werden.
```
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Beatmap/Title_text/de.md:55]: That is what I mean, but I am a bit confused now. The `hold` documentation says `How long the line will display before starting to fade out, in milliseconds`. This seems pretty clear (it is **not** `The time in the song when this line will start to fade out`, so I am positive this is meant in a relative context).

However, the default value is stated as `3600 + 800n`. In this case, `n = 2`, so it should be `3600 + 800 * 2`, no? Not sure where the `4000` comes from, and I don't see any interval of 400 that would be relevant here.

[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Beatmap/Title_text/de.md:55]: Judging from the last part (`after 2000ms pass`), this seems to be meant as

> 2000 ms später

and not

> nachdem 2000 ms vergangen sind (von t = 0 bzw. dem Start der Beatmap ausgesehen)

Thus, my question may be closed.

File: wiki/Beatmap/Title_text/de.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+# Beatmap-Titeltext
+
+![Screenshot von osu!-Gameplay mit sichtbarem Titeltext](img/liquid-title-text.jpg "Der Titeltext für \"Rostik - Liquid (Paul Rosenthal Remix)\" wird kurz nach dem Start der Beatmap angezeigt.")
+
+Der **Titeltext** einer Beatmap erscheint beim Spielen oben und mittig auf dem Bildschirm. Meistens zeigt der Text den Künstler und Titel des Songs kurz nach Beginn der Beatmap an, allerdings kann der Mapper den Inhalt des Textes beliebig anpassen. Auch die Formatierung und der Anzeigezeitpunkt des Titeltextes können geändert werden.
+
+Richtig eingesetzt kann der Titeltext einen schönen Effekt zur Beatmap hinzufügen. Vor allem Mapper, die [Storyboards](/wiki/Storyboard) verwenden, könnten es nützlich finden, den Text anzupassen, um die Darstellung der Beatmap während des Gameplays so genau wie möglich zu kontrollieren. Einige gute Beispiele für angepasste Titeltexte sind [Noisestorm - Solar](https://osu.ppy.sh/beatmapsets/33483), [IOSYS - Kanbu de Todomatte Sugu Tokeru ~ Kyouki no Udongein](https://osu.ppy.sh/beatmapsets/1391) und [Amane - Yume goro mo, Kinou no Koto -Rolling Contact Remix-](https://osu.ppy.sh/beatmapsets/57560).
+
+## Den Titeltext anpassen
```
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Beatmap/Title_text/de.md:16]: Seems I was behind the times. It was already caught by CI

**Review by The-Last-Cookie — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**